SharkCage/CageManager/CageManager.cpp

```diff
@@ -23,7 +24,33 @@ int main()
 	CageManager cage_manager;
 
 	SecuritySetup security_setup;
-	auto security_attributes = security_setup.GetSecurityAttributes();
+	//randomize the group name
+	UUID uuid;
```

Comment:
the UUID generation is now present more than once in this file, maybe you can relocate this to a function?